src/components/payment-options.js

```diff
@@ -94,7 +93,6 @@ var StripeButton = React.createClass({
           <div className={className}>
             <p>&nbsp;</p>
           </div>
-          <div className="row medium-label-size less-text">{this.context.intl.formatMessage({id: 'credit_card'})}</div>
```

Comment:
How about we keep this string hidden, instead of removing it? I think it’s the only way for screen readers to understand what the button is about